samples/docview/view.cpp

```diff
@@ -207,7 +207,9 @@ MyCanvas::MyCanvas(wxView *view, wxWindow *parent)
     m_currentSegment = nullptr;
     m_lastMousePos = wxDefaultPosition;
 
+#ifndef __WXQT__
```

Comment:
Could we avoid this, if necessary by just making `SetCursor()` do nothing in this case?

Reply:
Sorry I don't understand what you mean by this ?

Reply:
I'd like to avoid Qt-specific checks in the sample. I guess this is needed because `SetCursor()` doesn't work (in this case) currently?